marc_export/src/main/java/whelk/export/marc/MarcCliExport.java

```diff
@@ -237,6 +281,16 @@ private PreparedStatement getAllHeldURIsStatement(ExportProfile profile, Connect
         return preparedStatement;
     }
 
+    private PreparedStatement getAllSaoStatement(Connection connection)
+            throws SQLException
+    {
+        String sql = "SELECT id FROM lddb WHERE data#>>'{@graph,1,inScheme,@id}' = 'https://id.kb.se/term/saogf' AND deleted = false";
```

Comment:
That is SAOGF for GenreForm, is that what's intended here? SAO are the Subject headings.

Reply:
No, thats my mistake. I discussed it with Sara-Lean, but probably i misunderstood her. Whats the correct way to find the SAO subject headings?

Reply:
Fixed!

Reply:
Great!